File: app/src/main/java/com/sunny/youyun/activity/about_youyun/AboutYouyunContract.java
package com.sunny.youyun.activity.about_youyun;

import com.sunny.youyun.mvp.BaseModel;
import com.sunny.youyun.mvp.BasePresenter;
import com.sunny.youyun.mvp.BaseView;

/**
 * Created by Sunny on 2017/8/31 0031.
 */

interface AboutYouyunContract {
    interface View extends BaseView {

    }

    interface Model extends BaseModel {

    }

    abstract class Presenter extends BasePresenter<View, Model> {

    }
}

File: app/src/main/java/com/sunny/youyun/activity/about_youyun/AboutYouyunContract.kt
package com.sunny.youyun.activity.about_youyun

import com.sunny.youyun.mvp.BaseModel
import com.sunny.youyun.mvp.BasePresenter
import com.sunny.youyun.mvp.BaseView

/**
 * Created by Sunny on 2017/8/31 0031.
 */

interface AboutYouyunContract {
    interface View : BaseView

    interface Model : BaseModel

    abstract class Presenter : BasePresenter<View, Model>()
}
